jetbrains-rider/tst/software/aws/toolkits/jetbrains/services/lambda/completion/DotNetHandlerCompletionTest.kt

```diff
@@ -36,10 +35,9 @@ class DotNetHandlerCompletionTest : BaseTestWithSolution() {
         assertIconPath(handlers.first().iconId, "/resharper/PsiSymbols/Method.svg")
     }
 
-    @Test
+    // TODO this test only works on 2019.2. Which we don't support anymore. Fix the test
```

Comment:
How do we get this test fixed?

Reply:
No idea, it has been broken for a long time. I can take a stab at fixing it